schema_editor/app/scripts/schemas/schemas-service.js

```diff
@@ -0,0 +1,27 @@
+
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Schemas() {
+        var module = {
+            Object: Object
+        };
+        return module;
+
+        function Object() {
+            return {
+                type: 'object',
+                title: '',
+                plural_title: '',
```

Comment:
Yeah we could, theres a plugin out there and a few examples. Maybe as a separate task down the line? Not too worried about it now.